cli/exp_scaletest.go

```diff
@@ -974,6 +1004,7 @@ func (r *RootCmd) scaletestWorkspaceTraffic() *clibase.Cmd {
 					WriteMetrics: metrics.WriteMetrics(ws.OwnerName, ws.Name, agentName),
 					SSH:          ssh,
 					Echo:         ssh,
+					App:          appConfig,
```

Comment:
nit: we don't intend to scaletest 2 or more applications?

Reply:
Yeah, that's the state of things currently. If you want to test multiple things you can run multiple instances of `coder exp scaletest workspace-traffic`, e.g. one for ssh, one for web terminal and one for app.